docs/legacy/secure-communication-agents.asciidoc

```diff
@@ -106,7 +106,7 @@ POST /_security/role/apm_agent_key_role
   "cluster": ["manage_own_api_key"],
   "applications": [{
     "application": "apm",
-    "privileges": ["event:write", "config_agent:read"],
+    "privileges": ["event:write", "config_agent:read", "sourcemap:write"],
```

Comment:
And this fixes the 🐛 in https://www.elastic.co/guide/en/apm/guide/8.2/api-key-legacy.html